packs/slack/sensors/slack_sensor.py

```diff
@@ -180,6 +180,7 @@ def _handle_message(self, data):
                 'is_group': channel_info.get('is_group', False),
             },
             'timestamp': int(float(data['ts'])),
+            'timestamp_raw': data['ts'],
```

Comment:
On a somewhat related note - there is also a `number` type in json schema which represents numbers which can also hold floats.

This might come handy in some cases.